application/commands/TwigCommand.php

```diff
@@ -213,4 +269,24 @@ private function renderSurveyThemeManifest($oTemplateForPreview)
 
         $this->aLogs[$oTemplateForPreview->sTemplateName]['manifest'] =  "done";
     }
+
+    /**
+     * Finds all subfolders of $questionThemeDirectory containing XML files
+     * @param string $questionThemeDirectory
+     * @return array
+     */
+    private function getQuestionThemePaths($questionThemeDirectory)
+    {
+        $questionDirectoriesAndPaths = [];
+
+        $directory = new RecursiveDirectoryIterator($questionThemeDirectory);
+        $iterator = new RecursiveIteratorIterator($directory);
+        foreach ($iterator as $info) {
+            if ($info->getFileName() == 'config.xml') {
+                $questionDirectoriesAndPaths[] = dirname($info->getPathname());
```

Comment:
Maybe check if it's a question theme ? config->metadata->type == `question_theme` 

Since other don't do it : not mandatory… can be merged like this